lib/util/miq-system.rb

```diff
@@ -396,8 +396,7 @@ def self.swapinfo
   #
   ##############################################################################################################################
   def self.disk_usage(file=nil)
-    file = nil if file.blank?
-    raise "file #{file} does not exist" unless File.exist?(file.to_s) || file.nil?
+    file = df_filesystem(file)
```

Comment:
The method name is kind of weird...can't think of something off the top of my head.  Thinking `normalize_parameters_for_df(file)` and make it private?